Makd.mak

```diff
@@ -525,6 +528,19 @@ $O/test-%.stamp: $O/test-%
 	$(call exec,$< $(ITFLAGS),$<,run)
 	$Vtouch $@
 
+# Documentation rules
+###################
+.PHONY: doc
+
+# harbored-mod flags
+HMODFLAGS ?= -o $(D) --project-name=$(JOB_NAME) --project-version=$(PKGVERSION)
```

Comment:
I would move this next to the definition of `HMOD`, and maybe keep `-o $D` (you don't need the `()` for one-letter variables :heart: make) in the `$(call exec,$(HMOD)...)`, as is more sort of a fundamental part of the build rule.

Also, now that I look at it, the `$(JOB_NAME)` is too jenkins specific, I think the default should be different, maybe just getting the root project directory name by default? Also, for the version I would use the output of `mkversion` without transformations to be adapted to Debian versions. Maybe you could define a `VERSION` and then use that to build `PKGVERSION`.